basis/formatting/formatting-docs.factor

```diff
@@ -78,8 +80,10 @@ HELP: printf
         "##12" }
     { $example
         "USING: formatting ;"
-        "1234 \"%+d\" printf"
-        "+1234" }
+        " 1234 \"%+d\\n\" printf"
```

Comment:
The example looks correct. The whitespace before "1234" on line 83 is not in a string, it's in factor code. It's just so that examples align. The last line (line 86, "+1234\n-1234\n 1234" ) show the results with the correct '+' or ' ' in front of the number.

Compare what happens when you click on the example in the doc and it pastes it to the listener with and without the space.. It's just comestic..
![formatting](https://user-images.githubusercontent.com/89208/26879297-e21fd04a-4b90-11e7-9583-bd51b0880fe3.png)
![formatting_unaligned](https://user-images.githubusercontent.com/89208/26879408-432de3cc-4b91-11e7-8944-b686ffe7a769.png)

Anyway, I ran help-lint and it passed.